yarn.lock

```diff
@@ -2,73 +2,17 @@
 # yarn lockfile v1
```

Comment:
How are all this packages being removed? :|

can you run: 
```
rm -rf node_modules; rm -rf packages/*/node_modules; rm packages/*/package-lock.json; rm packages/*/yarn.lock; rm -rf packages/*/dist; rm -rf ~/.yarn-cache; npm cache clean --force; yarn cache clean; rm yarn.lock; yarn install
```

Reply:
Can you check ? I think it's fixed